src/main/java/net/snowflake/ingest/utils/SubColumnFinder.java

```diff
@@ -20,11 +20,13 @@ static class SubtreeInfo {
     final int startTag;
     final int endTag;
     final String dotPath;
+    final List<String> path;
```

Comment:
nit: add tests to SubColumnFinderTest too? Feel free to do it with the testing gaps JIRA.

Reply:
Added to test gap [JIRA](https://snowflakecomputing.atlassian.net/browse/SNOW-1490151)